Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN apk add --no-cache openssl
 
 COPY initdb.d      /docker-entrypoint-initdb.d
 COPY entrypoint.sh /usr/local/bin/entrypoint.sh
+COPY migratedb.d   /migratedb.d
```

Comment:
What happens if this file doesn't exist? Or it will always be there from now on?

Reply:
Ah, it's a folder, not a file. Sorry :)

Reply:
Is fine now or need extra comment for clarity?